segment-gcp.go

```diff
@@ -95,7 +94,7 @@ func getGCPProjectFromFile() (string, error) {
 		b = b[:coreEnd]
 	}
 
-	lines := bytes.Split(b[coreStart+len(gcloudCoreSectionHeader):coreEnd], []byte("\n"))
+	lines := bytes.Split(b[coreStart+len(gcloudCoreSectionHeader):], []byte("\n"))
```

Comment:
We cannot use `-1` as an array index in golang.
Also, if the case requires processing, it is truncated in the previous line, so processing should not be necessary here.